kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -378,14 +374,17 @@ class ResidualBasedBlockBuilderAndSolver
         else
             norm_b = 0.00;
 
-        if (norm_b != 0.00) {
+        if (norm_b != 0.00)

```

Comment:
This is a comestic change and does not follows the styling guide { in the same line